test_regress/t/t_empty_module.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
```

Comment:
Please rename these to t_mod_empty - we try to group test categories by first word.